opds_import.py

```diff
@@ -460,7 +461,9 @@ def __init__(self, _db, collection, data_source_name=None,
             # If this Collection is configured to mirror the assets it
             # discovers, this will create a MirrorUploader for that
             # Collection. Otherwise, this will return None.
-            mirror = MirrorUploader.for_collection(collection)
+            mirror = MirrorUploader.for_collection(
+                collection, ExternalIntegrationLink.COVERS
+            )
```

Comment:
This is a great payoff -- it's totally clear what's happening here.

However, it's also clear that `mirror` now is just for the book covers. We're going to need a separate `MirrorUploader` to store any ebooks we find. In this code, instead of self.mirror we should be using self.cover_mirror or self.book_mirror.

Reply:
Or we should have a dictionary that maps a purpose to a MirrorUploader designed for that purpose.